content/docs/reference-javascript-environment-requirements.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: javascript-environment-requirements
-title: JavaScript Environment Requirements
+title: JavaScript Ortam Gereksinimleri
 layout: docs
 category: Reference
 permalink: docs/javascript-environment-requirements.html
 ---
 
-React 16 depends on the collection types [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). If you support older browsers and devices which may not yet provide these natively (e.g. IE < 11) or which have non-compliant implementations (e.g. IE 11), consider including a global polyfill in your bundled application, such as [core-js](https://github.com/zloirock/core-js) or [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
+React 16 [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) ve [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) gibi koleksiyon tiplerine bağlıdır. Eğer bunları henüz yerel olarak sağlayamayan (ör. IE <11) veya uyumlu olmayan uygulamaları (ör. IE 11) olan eski tarayıcıları ve cihazları destekliyorsanız, paketlenmiş uygulamanıza global polyfill dahil etmeyi düşünün. [core-js](https://github.com/zloirock/core-js) ya da [babel-polyfill](https://babeljs.io/docs/usage/polyfill/) gibi.
 
-A polyfilled environment for React 16 using core-js to support older browsers might look like:
+React 16 için eski tarayıcıları destekleyen core-js kullanılan polyfilled bir ortam şöyle görünebilir:
```

Comment:
Suggestion:

> Eski tarayicilari desteklemek icin `core-js` kullanan polyfilled bir React 16 ortami soyle gorunebilir:

This is the way I prefer to interpret this sentence. We can discuss this, though.

Reply:
I've had trouble in that sentence. But your version is better. We can use it.

Reply:
Thanks for the fast response. Merging it...